lib/rules/curly.js

```diff
@@ -178,5 +237,8 @@ module.exports = function(context) {
 module.exports.schema = [
     {
         "enum": ["all", "multi", "multi-line", "multi-or-nest"]
+    },
+    {
+        "enum": ["consistent"]
```

Comment:
I think this is not quite right. `consistent` should not be allowed with `all`, as it doesn't make sense. Right?